.changeset/itchy-eyes-smoke.md

```diff
@@ -0,0 +1,8 @@
+---
+'@iota/graphql-transport': minor
+'@iota/iota-sdk': minor
+'@iota/apps-ui-kit': minor
+'@iota/kiosk': minor
+---
+
+Add `getSupportedIotaSystemState`
```

Comment:
We should also mention that this deprecated `getLatestIotaSystemState` in favor of `getLatestIotaSystemStateV2`